src/excmds.ts

```diff
@@ -2880,13 +2880,12 @@ import * as hinting from "./hinting"
         - -c [selector] hint links that match the css selector
           - `bind ;c hint -c [class*="expand"],[class="togg"]` works particularly well on reddit and HN
         - -w open in new window
-            -wp open in new private window
-        - `-pipe selector key` e.g, `-pipe * href` returns the key. Only makes sense with `composite`, e.g, `composite hint -pipe * textContent | yank`.
-        - `-W excmd...` append hint href to excmd and execute, e.g, `hint -W exclaim mpv` to open YouTube videos. Use `composite hint -pipe | [excmd]` instead.
-        - -q* quick (or rapid) hints mode. Stay in hint mode until you press <Esc>, e.g. `:hint -qb` to open multiple hints in the background or `:hint -qW excmd` to execute excmd once for each hint
+        - -wp open in new private window
+        - `-pipe selector key` e.g, `-pipe * href` returns the key. Only makes sense with `composite`, e.g, `composite hint -pipe * textContent | yank`. If you don't select a hint (i.e. press <Esc>), will return undefined.
```

Comment:
I think I'd rather return an empty string / abort somehow, as coercion to text will literally put "undefined" in the clipboard, etc.

e.g. `hint -pipe | tabopen` shouldn't open a blank tab when you hit escape.

Reply:
We currently can't abort piped commands unless we throw an exception and these will be caught and displayed in the command line. We probably don't want to do that everytime the user just leaves hint mode without having selected anything, so I made it return an empty string.